myko_api_client/setups/base_matcher_setup.py

```diff
@@ -0,0 +1,15 @@
+from time import sleep
+
+from myko_api_client.base_matcher import BaseMatcher
+
+
+class MykoMatcher(BaseMatcher):
+    pass
+
+import os
+os.environ["API_CLIENT_USERNAME"] = "dev@gridsingularity.com"
+os.environ["API_CLIENT_PASSWORD"] = "8gcBB9D5sK"
```

Comment:
Very bad! This looks like an actual password. Please remove. Ideally also from the commit history!